bin/validate.py

```diff
@@ -0,0 +1,65 @@
+from operator import truediv
+from os import listdir, getcwd, path
+import yaml, re
+
+def try_get_value(chart: dict, key: str) -> str:
+    try:
+        value = chart[key]
+    except KeyError:
+        return ""
+    return value
+
+def fail(value) -> bool:
+    print(f">    Error: Chart value '{value}' is incorrect or not set!")
+    return True
+
+def validate_chart_file(chart_file) -> bool:
+    failed = False
+    chart = yaml.safe_load(chart_file)
+    print(f"> Validating chart {chart['name']}...")
+
+    home = try_get_value(chart, "home")
+    description = try_get_value(chart, "description")
+    version = try_get_value(chart, "version")
+    icon = try_get_value(chart, "icon")
+
+    maintainers = try_get_value(chart, "maintainers")
+    try:
+        maintainers = maintainers[0]
+    except IndexError:
+        maintainers = {}
+    maintainer_name = try_get_value(maintainers, "name")
+    maintainer_email = try_get_value(maintainers, "email")
+
+    if home != "https://github.com/citizensadvice/helm-charts":
+        failed = fail("home")
+    
+    if description == "":
+        failed = fail("decription")
+
+    if maintainer_name != "CA Devops":
+        failed = fail("maintainer name")
+    
+    if maintainer_email != "ca-devops@citizensadvice.org.uk":
+        failed = fail("maintainer email")
+
+    if not re.match("^\d{1,3}.\d{1,3}.\d{1,3}$", version):
```

Comment:
regex here seems quite strict, did we agree on a versioning schema for all our charts?

Reply:
Do you think so? This is actually pretty loose compared to the regexs for semantic versioning I've seen others use. How would you change it?

Reply:
Yes it's just for the chart version, I've chosen not to check the app version as different teams will be using different schemes.